LayoutTests/http/tests/contentextensions/resources/iframe-that-opens-popup.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<body>
+    <script>
+        if (window.testRunner) {
+            const hostname = window.location.hostname;
+            alert(`Iframe loaded in ${hostname}. Attempting to open popup...`);
+            try {
+                const popup = window.open("about:blank");
+                if (!popup) {
+                    alert(`PASS: window.open() in ${hostname} returned null as expected.`);
```

Comment:
This is a good negative test that verifies that something was blocked.  Could we add a positive test to verify we can window.open a URL without about in it?